code/_onclick/click_handler.dm

```diff
@@ -169,6 +169,10 @@ var/const/CLICK_HANDLER_ALL                  = (~0)
 //If its not valid, null will be returned
 //In the case of click catchers, we resolve and return the turf under it
 /datum/click_handler/proc/resolve_world_target(var/a, var/params)
+	if(isliving(user))
```

Comment:
this isn't the place to do this, far too low level

click handlers are generic and used by many things, this problem lies specifically with kinesis

in addition, don't check stat directly, use the incapacitated() proc with the appropriate flag set from defines/mob.dm
https://i.imgur.com/X3GbK9v.png

probably knockout is what you need

Lastly, this is failing a map check, not sure why